src/DXRMapFixups/DeusEx/Classes/DXRFixupM03.uc

```diff
@@ -711,6 +719,11 @@ function AnyEntryMapFixes()
             }
         }
         break;
+    case "03_NYC_UNATCOHQ":
+        FixConversationFlagJump(GetConversation('AnnaAtUNATCO'), 'AnnaThanks_Played', false, 'AnnaThanksChatDone', false);
+        break;
+    case "03_NYC_HANGAR":
+        dxr.flagBase.SetBool('MeetLebedev2_Played', dxr.flagBase.GetBool('MeetLebedev2_Played'),, 5); // to enable some Paul dialog in M04
```

Comment:
There's no guarantee that the player will pass through Hangar on the way from the 747 back to HQ, due to Entrance Rando.

This shouldn't be done this way anyway though.  The conversation with Lebedev should set a flag with the appropriate expiry, which should then be checked in the later conversation (as previously discussed)